### ISSUE TITLE
With ACK set true, Devices returning error.

### ISSUE THREAD

**Issue by khawajamechatronics** [open-electronics/LoRa#4]
Hi,

I am trying to implemt the polling server and polled device example with ACK true. But its giving error 
int nc=LR.receiveNextMessage(NODEADD,dest,recbuff,bufflen,200);
nc==0 
so it seems the windows is too short or its RX windows is not turned ON on right time.

Can you check this issue?

**Comment by danieledenaro** [comment]
Yes. I agree, 200 millisecond is a fixed value. That is the problem! The solution is simple: I have to change the function "setAutomaticAck(bool set){autoAK=set;}" with new format "setAutomaticAck(boll set, int millisec){autoAK=set; AkTime=millisec;}". Where AkTime is a new variable (default 200). Obviously it must be update also function "writeMessage and writeMessageByte", substituting 200 with AkTime.
I will do it in the next release, but if you like you can change your dawnloaded library in this way. 
Thanks for warning.
